redaxo/src/core/lib/util/pager.php

```diff
@@ -80,7 +80,8 @@ public function getRowsPerPage()
     public function getCursor($pageNo = null)
     {
         if (null === $pageNo) {
-            $cursor = rex_request($this->cursorName, 'int', 0);
+            // BC: Fallback to "start"
+            $cursor = rex_request($this->cursorName, 'int', rex_request('start', 'int', 0));
```

Comment:
Der BC-Fallback ist ja jetzt direkt in der Pager-Klasse, die auch im Frontend verwendet wird.
Wäre es nicht besser, den Workaround nur in `rex_list` zu platzieren?
Zum Beispiel, indem man `$_REQUEST[$pager->getCursoName()] = rex_request('start', 'int', 0)` setzt (wenn nicht schon gesetzt)?

Reply:
Ich vermute dass es auch frontend logiken gibt die wir durch „umbennen“ des parameters beeinflusst haben, daher der fix hier.

Ausserdem wollte ich ungern in den superglobalen variablen schreibend eingreifen

Reply:
Überzeugt ;-)

https://github.com/redaxo/redaxo/pull/1844